pkg/loop/internal/relayer/pluginprovider/contractreader/contract_reader.go

```diff
@@ -624,6 +624,8 @@ func convertLimitAndSortToProto(limitAndSort query.LimitAndSort) (*pb.LimitAndSo
 			tp = pb.SortType_SortTimestamp
 		case query.SortBySequence:
 			tp = pb.SortType_SortSequence
+		case *query.SortBySequence:
+			tp = pb.SortType_SortSequence
```

Comment:
🤔 Do we need this? We don't support pointer based sortTypes for the other options in this list.

Reply:
Without this, we were getting the error "Unknown sort by type", when testing out full workflows

Reply:
Yes, this is for the logeventtrigger capability, when run as a LOOPP gRPC process

Reply:
This was the only type required for log event trigger and adding this works for our use case. We thought, other types could be added as needed.

Reply:
Can we spend some time understanding why we are making this change, and then fix it properly? (i.e. what, if anything, has changed upstream that requires this? What is the right way to use this API -- is it to use values, pointers, both? And if so, can we fix it for all supported values?)

Just adding the value you need right now is just asking us to change this later when someone starts to use other values.

Reply:
I did a bit of digging on this:

The answer is that query.LimitAndSort.SortBy is an interface type -- and although all existing implementations are values (SortByTimestamp, SortByBlock ...), the interfaces is also satisfiable by the equivalent pointer (since Go does implicit receiver conversions).

So the right fix here is to accept both values and pointers for all items in the enum.

Reply:
You should also put the two cases on the same line.  Ie 

case query.SortBySequence, *query.SortBySequence: